gatsby-config.js

```diff
@@ -214,5 +209,6 @@ module.exports = {
                 analyzerMode: 'static',
             },
         },
+        'gatsby-plugin-offline',
```

Comment:
we need to handle pushwoosh sw, please append it to workbox config

Reply:
All done now.